Transbank/Common/Options.cs

```diff
@@ -46,12 +32,11 @@ public IIntegrationType IntegrationType
                 );
         }
 
-        public Options(string commerceCode, string apiKey, IIntegrationType integrationType, RequestServiceHeaders headers)
+        public Options(string commerceCode, string apiKey, IIntegrationType integrationType)
```

Comment:
```suggestion
        public Options(string commerceCode, string apiKey, IIntegrationType integrationType, RequestServiceHeaders headers = null)
```

What if we keep that parameter but as optional?

Reply:
I think that would not be a breaking change.